docs/configuration.rst

```diff
@@ -232,17 +232,23 @@ Executor Manager Properties
 Notification Email Properties
 ########
 
-+---------------+-----------------------------------------------------+---------+
-| Parameter     | Description                                         | Default |
-+===============+=====================================================+=========+
-| mail.sender   | The email address that azkaban uses to send emails. |         |
-+---------------+-----------------------------------------------------+---------+
-| mail.host     | The email server host machine.                      |         |
-+---------------+-----------------------------------------------------+---------+
-| mail.user     | The email server user name.                         |         |
-+---------------+-----------------------------------------------------+---------+
-| mail.password | The email password user name.                       |         |
-+---------------+-----------------------------------------------------+---------+
++---------------+-----------------------------------------------------+-----------+
+| Parameter     | Description                                         | Default   |
++===============+=====================================================+===========+
+| mail.sender   | The email address that azkaban uses to send emails. |           |
++---------------+-----------------------------------------------------+-----------+
+| mail.host     | The email server host machine.                      | localhost |
++---------------+-----------------------------------------------------+-----------+
+| mail.host     | The email server port.                              | 25        |
```

Comment:
Could you please fix the typo here? It should be `mail.port` right?

Reply:
Shoot, my bad, thanks for catching it. Fixed.